### PR TITLE
[HWKMETRICS-585] Merge dropwizard reporter in metrics

### PR DESCRIPTION
**Pull request by jotak**
And use hawkular-metrics-parent as new maven parent, adapt pom to remove items already managed in parent.

Note about travis: I've let the existing travis config, but I'm not sure if it's going to work as is since it's now  in a subfolder. Basically, this config pulls & start container for running metrics in order to execute integration tests. It was copied from the ruby client travis scripts.

### CODE REVIEW

**Comment by rhqci** [comment]
Can one of the admins verify this patch?

File: clients/common/pom.xml [hunk 1]
```diff
@@ -32,6 +32,19 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.glassfish</groupId>
```
Comment (jotak) [inline clients/common/pom.xml]: I'm not 100% sure if it's the right place for that (in clients/common, scope provided)... is it ok?

**Comment by jotak** [comment]
@stefannegrea I've switched to embedded hawkular/wildfly for integration tests

**Comment by jsanda** [comment]
Sorry for the delay in getting to this. Thanks for doing this @jotak!

**Pull request merged**